### PR TITLE
:art: Homepage improvements

### PR DESCRIPTION
**Pull request by thibmeu**
This issue help users answering the following questions:
+ What is the Workbench? What's its aim?
+ Who should/can use it?
+ Which content is covered?
+ How do you contribute content?
+ Which companies support it?
+ How can I donate to support further development?
+ Can I use this as part of my course (University, school, etc.)

What it looks like:
2019-05-07 12 48 51 localhost 721b87247f05

What has been done:
+ Partners
  + Logos updated with proper companies
  + Bigger title
  + Shadow on logos

+ Feature section
  + Right after the landing screen, features are presented in more detailed for specific set of users: programmers, universities, companies
  + Small picture and text explanation

+ Learning
  + Links updated and points to existing content

+ Landing screen
  + `/start` link updated, now points to valid content

+ Contribution
  + Contribution via Github or donation

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -39,60 +57,68 @@ export default function Home() {
...
               <div className="column feature-box">
-                <h1 className={'title'}>Smart contracts</h1>
+                <h1 className={'title'}>Start Learning</h1>
                 <aside className="menu">
                   <ul className="menu-list">
                     <li>
-                      <a href={'http://'} className="text is-2">
-                        Functions
-                      </a>
+                      <Link to={'/Intro/Merkle_Tree'} className="text is-2">
```
Comment (severus-sn4pe) [inline src/components/Home.js]: should be `<Link to={'/pages/Intro/Merkle_Tree'} className="text is-2">`

File: src/components/Home.js [hunk 2]
```diff
@@ -106,15 +132,49 @@ export default function Home() {
...
+        <div className={'container'}>
+          <h1 className={'title is-1'}>Contribute</h1>
+          <h2 className={'subtitle has-75-percent-width'}>
+            The project is made by engineers. Everything being open source, you can contribute directly to it by
+            providing documentation, translating it, or directly coding new features. You can also donate directly
+            towards the project.
+          </h2>
+          <div className={'buttons'}>
+            <a
+              href={'https://github.com/blockchainworkbench'}
```
Comment (severus-sn4pe) [inline src/components/Home.js:161]: adding `target={'_blank'} rel={'noopener noreferrer'}` here like on external links in footer would make sense

**Review by severus-sn4pe — APPROVED** [review]
looks good, some minor changes suggested

File: src/components/Home.js (hunk 1 above)
Comment (thibmeu) [reply src/components/Home.js]: fixed

File: src/components/Home.js (hunk 2 above)
Comment (thibmeu) [reply src/components/Home.js:161]: fixed for github and donation url

**Pull request merged**